### DISCUSSION TITLE
manual fan control does not work after thinkfan exiting

### DISCUSSION THREAD

**Discussion by alex-ie** [vmatare/thinkfan#168] [Support]
I use Linux Mint and for latest 20.2 apt gets thinkfan 0.9.1, so beg pardon if the issue is fixed in newest one. I'm not very good at building from source, but I think I'll try if indeed the issue is solved. So the issue as in the topic: manual fan control does not work after thinkfan exiting (echo level 0 | sudo tee /proc/acpi/ibm/fan does not result in permanent fan disabling). More details in https://unix.stackexchange.com/questions/677382/why-fan-gets-enabled-again-and-again-after-thinkfan-started-and-ended-once

**Comment by koutheir** [comment]
Can you reproduce this behavior with a binary built from the latest sources?

The version 0.9.1 is more than 5 years old at this point, so I doubt you could get support for it. @vmatare, please correct me if I'm wrong.

**Comment by alex-ie** [comment]
I've found the cause, I've found in `thinkfan` (at least 0.9) code (as I've understood it) activation of `watchdog` functionality of `thinkpad_acpi`, I've learned to disable it back. I don't mind deleting this topic, but I don't know the rules about such things here...

**Reply by vmatare** [reply]
You're correct, thinkfan enables the watchdog functionality of the `thinkpad_acpi` kernel module and it's certainly possible that it isn't disabled again after thinkfan exits. But what I don't understand is why you'd want to permanently switch off your laptop fan, without having any control in place that monitors temperatures...? To me that sounds like a sure-fire way to ruin your mainboard...

**Comment by koutheir** [comment]
> I don't mind deleting this topic, but I don't know the rules about such things here...

@vmatare, should this topic be deleted?

**Reply by vmatare** [reply]
As long as there's any potential benefit for other users I wouldn't delete it
